packages/custom-props-layer/src/components/map/Map.tsx

```diff
@@ -0,0 +1,34 @@
+import { ComponentProps } from "../../types";
+import { Label } from "../commons/Label";
+import { PropertyContainer } from "../commons/PropertyContainer";
+import { Text } from "../text/Text";
+import { usePageRoutes } from "../../hooks/usePageRoutes";
+
+export const Map: React.FC<ComponentProps> = (props) => {
+  const { routes } = usePageRoutes();
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <PropertyContainer>
+        <Label name={props.propName} />
+      </PropertyContainer>
+      {props.attributes!.map((attribute, index) => {
+        if (attribute.type === "text") {
```

Comment:
We already have similar code written in `TabBody.tsx` I guess. Move this JSX into a separate component so that you can reuse it here.

Reply:
Done.